src/main/java/com/ning/maven/plugins/dependencyversionscheck/AbstractDependencyVersionsMojo.java

```diff
@@ -407,66 +423,96 @@ protected Map buildResolutionMap(final String scope) throws MojoExecutionExcepti
 
         // Map from artifactName --> list of resolutions found on the tree
         final SortedMap resolutionMap = new TreeMap();
+        final List errors = new ArrayList();
+        boolean useParallelDepResolution = "true".equalsIgnoreCase(getPropertyOrDefault("dependency.version.check.useParallelDepResolution", "true"));
```

Comment:
This should be done as a normal Mojo parameter, so it can be configured in the POM. We can also give it a property as part of the Mojo parameter, so that it can be overridden on the Maven command line with `-D`.

http://books.sonatype.com/mvnref-book/reference/writing-plugins-sect-mojo-params.html